_episodes/docker-desktop.md

```diff
@@ -0,0 +1,411 @@
+
+{::options parse_block_html="true" /}
+<div>
+<ul class="nav nav-tabs nav-justified" role="tablist">
+  <li role="presentation" class="active">
+    <a href="#install_htop" aria-controls="install_htop" role="tab" data-toggle="tab">Installing htop</a>  </li>
+  <li role="presentation">
+    <a href="#htop_running" aria-controls="htop_running" role="tab" data-toggle="tab">Running htop</a>  </li>
+</ul>
+<div class="tab-content">
+  <article role="tabpanel" class="tab-pane active" id="install_htop">
+  ![Terminal, installing htop inside getting-started container.](../fig/docker-desktop/24_gettingstarted_htop.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="htop_running">
+  ![Terminal, running htop inside getting-started container.](../fig/docker-desktop/25_gettingstarted_htop2.png)
+  </article>
+</div>
+</div>
+<br/>
+{::options parse_block_html="false" /}
+
+The container does not need to stay alive forever, and you can see that there is a 'stop' icon on the top right.
+If we stop the container, we get a familiar empty tab in 'Terminal' and 'Stats'.
+The 'Containers' tab on the left will also show the container status as 'Exited'
+
+{::options parse_block_html="true" /}
+<div>
+<ul class="nav nav-tabs nav-justified" role="tablist">
+  <li role="presentation" class="active">
+    <a href="#gs_stopped_Terminal" aria-controls="gs_stopped_Terminal" role="tab" data-toggle="tab">Terminal</a>  </li>
+  <li role="presentation">
+    <a href="#gs_stopped_Stats" aria-controls="gs_stopped_Stats" role="tab" data-toggle="tab">Stats</a>  </li>
+  <li role="presentation">
+    <a href="#gs_stopped_Containers" aria-controls="gs_stopped_Containers" role="tab" data-toggle="tab">Containers</a>  </li>
+</ul>
+<div class="tab-content">
+  <article role="tabpanel" class="tab-pane active" id="gs_stopped_Terminal">
+  ![Terminal tab in container from stopped getting-started image.](../fig/docker-desktop/26_gettingstarted_exited_t.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_stopped_Stats">
+  ![Stats tab in container from stopped getting-started image.](../fig/docker-desktop/27_gettingstarted_exited_s.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_stopped_Containers">
+  ![Container list after stopping getting-started image.](../fig/docker-desktop/28_gettingstarted_exited_c.png)
+  </article>
+</div>
+</div>
+<br/>
+{::options parse_block_html="false" /}
+
+Now lets say I want to run the `getting-started` image again. So I go to the 'Images' tab, and click run.
+Now lets go to the 'Terminal' tab, and try and find our directory with `ls`. The directory is not there.
+We'd also installed `htop`. so lets have a go at running it. Not there either.
+![Terminal in fresh getting-started image.](../fig/docker-desktop/29_gettingstarted_2.png)
+
+If this is not surprising to you, it means you're getting the hang of it.
```

Comment:
Personally not a fan of this kind of thing as they make me feel silly if I'm not keeping up

Reply:
Removed...